wordpress-mobile/WordPress-iOS
# Jetpack section in My Site

Fixes #15275
 
This PR groups Stats, Activity log and Jetpack Settings in the Jetpack section. 
Jetpack settings have been extracted from site settings and now exist in both places. 

This PR also renames `JetpackSecuritySettingsViewController` to `JetpackSettingsViewController` 
and groups the the related classes under a new subfolder - Jetpack Settings

New Tracks:
jetpack_settings_viewed
jetpack_manage_connection_viewed
jetpack_disconnect_tapped
jetpack_disconnect_requested - (User selected disconnect in alert)


![jetpack_section](https://user-images.githubusercontent.com/1335657/99020991-8839e000-2514-11eb-9d9c-52eeab0a69df.gif)



To test:
1. Go to My Site
2. See the Jetpack Section 
3. See Jetpack Settings row only if you have a plan 
4. Test all new events are fired correctly 
5. Test it doesn't affect Quick Start 

PR submission checklist:
